generator/src/test/java/com/scottlogic/deg/generator/cucumber/After.feature

```diff
@@ -64,24 +62,22 @@ Scenario: 'After' with a non-contradicting 'Not After' is successful
   Then the following data should be generated:
     | foo                     |
     | 2019-01-01T00:00:00.001 |
-    | 2019-01-01T00:00:00.002 |
-    | 2019-01-01T00:00:00.003 |
-    | 2019-01-01T00:00:00.004 |
-    | 2019-02-01T00:00:00.005 |
+    | 2019-02-01T00:00:00.001 |
+    | 2019-03-01T00:00:00.001 |
+    | 2019-04-01T00:00:00.001 |
+    | 2019-05-01T00:00:00.001 |
 
-# Defect 611 "before run against a non contradicting before results in an error" related to this scenario
-@ignore
 Scenario: 'Not After' with a non-contradicting 'Not After' is successful
   Given foo is anything but after 2019-01-01T00:00:00.000
     And foo is anything but after 2020-01-01T00:00:00.000
     And the generator can generate at most 5 rows
   Then the following data should be generated:
     | foo                     |
-    | 2019-01-01T00:00:00.000 |
-    | 2018-12-31T23:59:59.999 |
-    | 2018-12-31T23:59:59.998 |
-    | 2018-12-31T23:59:59.997 |
-    | 2018-12-31T23:59:59.996 |
+    | 0001-01-01T00:00:00.000 |
+    | 0002-01-01T00:00:00.000 |
+    | 0003-01-01T00:00:00.000 |
+    | 0004-01-01T00:00:00.000 |
+    | 0005-01-01T00:00:00.000 |
```

Comment:
Tests should be set to expect the generation of dates in the order of before date down (rather than from 0001-01-01 up). This is to be in line with the proposed design.